scripts/tests_analyzer/compare_coderabbit_decisions.py

```
@@ -0,0 +1,689 @@
+    lines.append("")
+
+    return lines
+
+
+def generate_markdown_report(results: list[ComparisonResult], repo: str, *, detailed: bool = False) -> str:
+    """Generate a Markdown report of the comparison results."""
+    lines = [
+        "# CodeRabbit vs Pytest Marker Analyzer Comparison Report",
+        "",
+        f"**Repository:** [{repo}](https://github.com/{repo})",
+        f"**Generated:** {datetime.now(tz=timezone.utc).strftime('%Y-%m-%d %H:%M:%S')}",
```

Ruff (0.14.10)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)